app/models/storage_location.rb

```diff
@@ -56,6 +56,10 @@ def item_total(item_id)
     inventory_items.select(:quantity).find_by(item_id: item_id).try(:quantity)
   end
 
+  def total_inventory_fair_market_value
+    items.sum(:value_in_cents)
```

Comment:
Can you use a safe-nav on the `value_in_cents`  call? I'd be worried about the behavior if the value_in_cents is nil.